### PR TITLE
remove repeated code

### PR DESCRIPTION
**Pull request by ManojJiSharma**
## Description

Creating crate for same code in different crates to put shared testing logic.

[https://github.com/Analog-Labs/chain-connectors/pull/253#discussion_r1766777781](https://github.com/Analog-Labs/chain-connectors/pull/253#discussion_r1766777781)

### CODE REVIEW

File: chains/utils/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[package]
+name = "utils"
```
Comment (Lohann) [inline chains/utils/Cargo.toml:2]: This name is too generic, maybe `rosetta-ethereum-testing`

Comment (ManojJiSharma) [reply chains/utils/Cargo.toml]: updated with `rosetta-chain-testing`

File: chains/rosetta-chain-testing/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,33 @@
+use std::future::Future;
+
+/// Run the test in another thread while sending txs
+/// # Panic
+/// Panics if the future panics
+pub async fn run_test<Fut: Future<Output = ()> + Send + 'static>(future: Fut) {
```
Comment (Lohann) [inline chains/rosetta-chain-testing/src/lib.rs:6]: Just tokyo as depedency, nice!

Comment (Lohann) [inline chains/rosetta-chain-testing/src/lib.rs:6]: Just `tokio` as dependency, NICE! great job @ManojJiSharma !!

**Review by Lohann — APPROVED** [review]

**Pull request merged**